ophyd/sim.py

```diff
@@ -761,7 +764,7 @@ class TrivialFlyer:
     name = "trivial_flyer"
     parent = None
 
-    def kickoff(self):
+    def kickoff(self) -> Any:
```

Comment:
```suggestion
    def kickoff(self) -> NullStatus:
```
?

Reply:
I thought I had to change that to `Any` because the type checker didn't think that Ophyd's version of `BaseStatus` conformed to Bluesky's `Status` protocol but I'll double check